package.json

```
@@ -28,57 +24,39 @@
   },
   "files": [
     "LICENSE",
-    "scripts",
-    "typechain",
     "README.md",
-    "hardhat.config.ts",
-    "dist/**/!(tsconfig.tsbuildinfo)",
-    "artifacts/contracts/EtherSwap.sol/EtherSwap.json",
-    "artifacts/contracts/ERC20Swap.sol/ERC20Swap.json",
-    "artifacts/contracts/TestERC20.sol/TestERC20.json",
-    "artifacts/@openzeppelin/contracts/token/ERC20/ERC20.sol/ERC20.json"
+    "dist/**/!(tsconfig.tsbuildinfo)"
   ],
   "dependencies": {
     "@boltz/bitcoin-ops": "^2.0.0",
-    "bip32": "^3.0.1",
+    "bip32": "^4.0.0",
     "bip65": "^1.0.3",
     "bip66": "^1.1.5",
-    "bitcoinjs-lib": "^6.0.1",
-    "bn.js": "^5.2.0",
-    "ecpair": "^2.0.1",
-    "liquidjs-lib": "^6.0.2-liquid.2",
+    "bitcoinjs-lib": "^6.1.0",
+    "bn.js": "^5.2.1",
+    "ecpair": "^2.1.0",
```

```suggestion
    "ecpair": "^2.0.1",
```
To avoid errors when used with blinding